docs/src/build_from_source.md

```diff
@@ -0,0 +1,48 @@
+# Building TensorFlow from source
+
+Building TensorFlow from source is recommended by Google for maximum performance - on some systems, the difference can be substantial. It will also be required for Mac OS X GPU support for versions later than 1.1. This document describes how to do so.
+
+
+## Step 1: build libtensorflow
+
+To build libtensorflow for TensorFlow.jl, follow the steps here: https://www.tensorflow.org/install/install_sources, except for a few minor modifications.
+
+  * Ignore the step "Install python dependencies".
```

Comment:
I suggest maybe:

 Ignore the step "Install python dependencies" -- these will not be needed as we are not building  for python. (You do still need to install Bazel though.)